src/screens/block/block.vue

```diff
@@ -113,10 +116,27 @@ export default {
       } else if (blockHashRegex.test(this.blockId)) {
         this.$store.dispatch('loadBlock', { hash: this.blockId })
       }
+    },
+    keyWatcher () {
+      let self = this
+      window.addEventListener('keyup', function (ev) {
+        self.blockNavigation(ev)
+      })
+    },
+    blockNavigation (ev) {
+      if (this.$route.name === 'Block') {
+        if (ev.keyCode === 39 && this.height > this.block.height) {
```

Comment:
this.height might not be set (or default value 0) if you open the url directly e.g. via `http://localhost:8080/#/block/9400`